kitsune/dashboards/templates/dashboards/localization.html

```diff
@@ -2,7 +2,7 @@
 {% from "dashboards/includes/macros.html" import print_readout, overview_section, product_choice_list, print_subscription_menu with context %}
 {% from "dashboards/includes/macros.html" import localization_sidebar_nav %}
 {% set title = _('[{locale}][{product}] Localization Dashboard')|f(locale=current_locale, product=_(product.title, 'DB: products.Product.title') if product else _('All Products')) %}
-{% set scripts = ('wiki', 'wiki.dashboard') %}
+{% set scripts = ('wiki', 'jqueryui', 'wiki.dashboard') %}
 {% set styles = ('kbdashboards', ) %}
```

Comment:
so the datepicker is working now but has no style. so you'll probably need to add 'jqueryui' here too.